src/flags/color.rs

```diff
@@ -89,6 +90,14 @@ impl Configurable<Self> for ColorOption {
             None
         }
     }
+
+    fn from_environment() -> Option<Self> {
+        if env::var("NO_COLOR").is_ok() {
```

Comment:
Should we be checking if the value is truthy? Should we sill be disabling colors if `NO_COLORS` is set to 0?

Reply:
> All command-line software which outputs text with ANSI color added should check for the presence of a NO_COLOR environment variable that, when present (**regardless of its value**), prevents the addition of ANSI color.
https://no-color.org/
If someone doesn't want `NO_COLOR` then they should just unset it.